packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -507,8 +508,32 @@ export interface CommonProjectProps {
    * The physical, human-readable name of the CodeBuild Project.
    */
   projectName?: string;
-}
 
+  /**
+   * VPC network to place codebuild network interfaces
+   *
+   * Specify this if the codebuild project needs to access resources in a VPC.
+   */
+  vpc?: ec2.IVpcNetwork;
+
+  /**
+   * Where to place the network interfaces within the VPC.
+   *
+   * Only used if 'vpc' is supplied.
+   *
+   * @default All private subnets
+   */
+  vpcPlacement?: ec2.VpcPlacementStrategy;
+
+  /**
+   * What security group to associate with the codebuild project's network interfaces.
+   *
+   * Only used if 'vpc' is supplied.
+   *
```

Comment:
@default?

Reply:
If a VPC is defined and this is not supplied, an implicit SecurityGroup should be created and it should be accessible through `project.securityGroups`

Reply:
> @default?

Following what I believed to be precedence in other modules.